Gems/Atom/Feature/Common/Code/Source/PostProcessing/LookModificationCompositePass.cpp

```diff
@@ -132,7 +132,7 @@ namespace AZ
             if (scene)
             {
                 PostProcessFeatureProcessor* fp = scene->GetFeatureProcessor<PostProcessFeatureProcessor>();
-                AZ::RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetDefaultView();
+                AZ::RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetFirstView(GetPipelineViewTag());
```

Comment:
Maybe irrelevant to your PR, it probably shouldn't use scene->GetDefaultRenderPipeline() but use m_pipeline (or GetRenderPipeline()